src/test/java/com/google/alpollo/DataBaseTest.java

```diff
@@ -0,0 +1,121 @@
+package com.google.alpollo;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.HashMap;
+import com.google.alpollo.model.Song;
+import com.google.alpollo.model.SongCounter;
+import com.google.alpollo.model.AnalysisInfo;
+import com.google.alpollo.SongDataBase;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.googlecode.objectify.ObjectifyFactory;
+import com.googlecode.objectify.ObjectifyService;
+import com.googlecode.objectify.cache.AsyncCacheFilter;
+import com.googlecode.objectify.util.Closeable;
+import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.After;
+
+@RunWith(JUnit4.class)
+public final class DataBaseTest {
+  private final LocalServiceTestHelper helper = new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  private Closeable session;
+
+  private static final Song SONG = new Song("Drake", "Toosie Slide");
+  /** We save 10 songs to database and to test it we use this constant. */
+  private static final int TEN = 10;
+  /** 
+   * We save 10 songs to database and this constant provides us to check adding more then 10 songs, 
+   * in this case 15, but we can change. 
+   */
+  private static final int NUM_OF_ACCEPTED_SONGS = 15;
```

Comment:
Instead of setting a fixed number, it would be better to define it relatively to the constant above, e.g. `NUM_OF_ACCEPTED_SONGS = NUM_SONGS_DATABASE + 5`.